Package.swift

```diff
@@ -67,10 +67,8 @@ let package = Package(
         ],
         products: [
             .library(name: "Purchases",
-                    targets: ["Purchases"]),
```

Comment:
Trailing commas in lists are great for preventing diff noise. If the linter removed it, I think we should adjust the rule.